test/functional/notifications/notify.js

```diff
@@ -1,118 +1,174 @@
-import assert from 'assert';
-import _ from 'lodash';
+import assert from "assert";
```

Comment:
you need to check what sort of formatting you are using, many places are affected. In this specific place, we use a single quote ( I wonder why lint is not caching this)

Reply:
fixed